course02/homework01/linked_list.h

```diff
@@ -0,0 +1,70 @@
+#include <iostream>
+
+using namespace std;
+
+class LinkedList
+{
+
+public:
+	LinkedList();
+	~LinkedList();
+
+	struct Node
+	{
+		Node* next{ nullptr };
+		int data;
+	};
+
+	void Insert(int key);
```

Comment:
Insert isn't the best method name, as we don't know where exactly. Insert is typically used as a method to insert an element at a specific position. Otherwise, you would name it prepend/append or push_front/push_back